data/samyabrata-maji.json

```
@@ -1,28 +1,49 @@
 {
   "name": "Samyabrata Maji",
   "type": "personal",
-  "bio": "Student • Web developer  • Open Source Enthusiast • UI/UX designer",
+  "displayStatsPublic": true,
+  "bio": "Student • Web developer  • Open Source Enthusiast • UI/UX designer • Technical Blogger",
   "avatar": "https://avatars.githubusercontent.com/u/116789799?v=4",
+  "tags": [
+    "Web Development",
+    "React",
+    "JavaScript",
+    "Open Source",
+    "Java",
+    "Python",
+    "Kotlin"
+  ],
+  "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/samyabrata-maji" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/sammaji15" },
+    { "icon": "FaLinkedin", "url": "https://linkedin.com/in/samyabrata-maji" },
+    { "icon": "FaInstgram", "url": "https://instagram.com/sammaji15" }
+  ],
   "links": [
     {
-      "name": "GitHub",
+      "name": "Checkout my code contributions on GitHub",
       "url": "https://github.com/samyabrata-maji",
       "icon": "FaGithub"
     },
     {
-      "name": "Twitter",
-      "url": "https://twitter.com/samaji15",
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sammaji15",
       "icon": "FaTwitter"
     },
     {
-      "name": "Blogs",
+      "name": "Read my Blogs on Hashnode",
       "url": "https://sammaji.hashnode.dev",
       "icon": "FaHashnode"
     },
     {
-      "name": "LinkedIn",
+      "name": "Connect with me on LinkedIn",
       "url": "https://linkedin.com/in/samyabrata-maji",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/sammaji15",
+      "icon": "FaLinkedin"
```

Not sure if you wanted to use this icon.
```suggestion
      "icon": "FaInstagram"
```

---

My Mistake